.gitignore

```diff
@@ -142,7 +142,7 @@ pnpm-lock.yaml
 yarn.lock
 
 # editor files
-.vscode
+# .vscode
```

Comment:
Are we sure we should remove this? What if the user edits these files?

Reply:
I've put it back @mcollina